lib/browser/chrome-extension.js

```diff
@@ -232,7 +232,7 @@ const getMessagesPath = (extensionId) => {
   }
 }
 
-ipcMainUtils.handleSync('CHROME_GET_MESSAGES', function (event, extensionId) {
+ipcMainUtils.handle('CHROME_GET_MESSAGES', function (event, extensionId) {
   const messagesPath = getMessagesPath(extensionId)
   return fs.readFileSync(messagesPath)
```

Comment:
Not required in this PR, but worth noting this can become `async`

Reply:
I've actually noticed that and have a PR https://github.com/electron/electron/pull/17148 :)